### PR TITLE
feat(todomvc-server): switch to typeorm and vesper

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: packages/apps/todomvc-server/src/controller/TodoController.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Controller, Query, Mutation } from 'vesper'
+import { Todo } from '../model/Todo'
+
+@Controller()
+export class TodoController {
+  @Query()
+  queryTodos() {
```
Comment (gaspard) [inline packages/apps/todomvc-server/src/controller/TodoController.ts]: Can we rename to `queryTodo` ? I like the fact that we simply append the type to 'query', 'mutate', 'subscribe' without having to deal with pluralization (which is a mess). This allows us to potentially write the GraphQL automatically.

File: packages/apps/todomvc-server/src/controller/TodoController.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    if (args._deleted) {
+      if (todo) {
+        await Todo.delete({ id: args.id })
+      } else {
+        throw new Error("Todo to remove doesn't exists")
+      }
+      return todo
+    } else {
+      if (!todo) {
+        todo = await Todo.create(args)
```
Comment (gaspard) [inline packages/apps/todomvc-server/src/controller/TodoController.ts]: I think it should have a `return todo` here...

File: packages/apps/todomvc-server/src/schema/controller/TodoController.graphql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+type Query {
+  queryTodos: [Todo!]!
```
Comment (gaspard) [inline packages/apps/todomvc-server/src/schema/controller/TodoController.graphql]: Same here: `queryTodo`

File: packages/apps/todomvc-server/src/controller/TodoController.ts (hunk 1 above)
Comment (henri-hulski) [reply packages/apps/todomvc-server/src/controller/TodoController.ts]: Sure. Will change.

File: packages/apps/todomvc-server/src/controller/TodoController.ts (hunk 2 above)
Comment (henri-hulski) [reply packages/apps/todomvc-server/src/controller/TodoController.ts]: It returns on save.
`save` actually either creates a new `Todo` or update the existent.
`preload` loads the existing item and merge with the `args`. When not exist it returns `undefined.
In that case I create a new `Todo` instance. Either of them is saved after.

File: packages/apps/todomvc-server/src/schema/controller/TodoController.graphql (hunk 3 above)
Comment (henri-hulski) [reply packages/apps/todomvc-server/src/schema/controller/TodoController.graphql]: Ok

**Pull request merged**